packages/mobile/src/services/env.ts

```diff
@@ -0,0 +1,3 @@
+export const env = {
+  EAGER_DISCOVERY_NODES: process.env.REACT_APP_EAGER_DISCOVERY_NODES
+}
```

Comment:
It probably makes sense to refactor audiusBackend to use this too. But can do that later